express/parsers/apps/espresso/parser.py

```diff
@@ -331,3 +332,10 @@ def final_structure_strings(self):
             except:
                 pass
         return structures
+
+    def averaged_potential(self):
+        data = self.txt_parser.averaged_quantity()
+        data["x"] *= Constant.BOHR  # convert to angstrom
+        data["p_x"] *= Constant.RYDBERG  # convert to eV
```

Comment:
maybe just planar_average and macroscopic_average

Reply:
I used the same naming as in https://gitlab.com/QEF/q-e/blob/develop/PP/src/average.f90, but since we will expose this array to the user I'll rename the keys